web/src/containers/IncentivePayments.js

```diff
@@ -0,0 +1,142 @@
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+
+import { updateApd as updateApdAction } from '../actions/apd';
+import { Input, DollarInput } from '../components/Inputs';
+import { t } from '../i18n';
+import { INCENTIVE_ENTRIES } from '../util';
+import { formatMoney, formatNum } from '../util/formats';
+
+const QUARTERS = [1, 2, 3, 4];
+const COLORS = {
+  '2018': 'bg-teal',
+  '2019': 'bg-green'
```

Comment:
😬 hardcoded years!  Could this just be an array and have it map to the index of the years array instead of being tied to specific years?  I think the second argument to the `Array.map` callback is the index?  (Would have to look it up, but that seems right.)

Reply:
good call, this is gross!